Frameworks/include/ABSourceInternal.h

```diff
@@ -0,0 +1,28 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#pragma once
+
+#import "UWP/WindowsApplicationModelContacts.h"
+#import "ABRecordInternal.h"
+#import <AddressBook/ABRecord.h>
+
+@interface _ABSource : _ABRecord
+
+@property ABRecordID recordID;
+- (id)initWithRecordID:(ABRecordID)recordID;
```

Comment:
Bringing up a separate concern on the .mm, but if that's the case, can we leave this portion of the change until we also fill in ABPersonCopySource? As it is, it's currently a solution without a problem.

---
In reply to: [74813579](https://github.com/Microsoft/WinObjC/pull/881#discussion_r74813579) [](ancestors = 74813579,74811477)